nautobot/core/tests/test_forms.py

```diff
@@ -692,7 +692,6 @@ def test_dynamic_filter_form(self):
                 site_form._get_lookup_field_choices(),
```

Comment:
This whole test will need to be updated to use something other than `site` as its basis, at some point.